src/server/middleware/sendHtml.js

```diff
@@ -69,30 +89,34 @@ export function renderStaticErrorPage(res) {
     message = 'Sorry, we are unable to load this page at this time.';
   }
 
-  // TODO: allow root module to provide custom error message and override default html
-  safeSend(res,
-    `<!DOCTYPE html>
-        <html>
-          <head>
-            <title>One App</title>
-            <meta http-equiv="X-UA-Compatible" content="IE=edge">
-            <meta charset="utf-8">
-            <meta name="viewport" content="width=device-width, initial-scale=1">
-            <meta name="application-name" content="one-app">
-          </head>
-          <body style="background-color: #F0F0F0">
-            <div id="root">
-              <div>
-                <div style="width: 70%; background-color: white; margin: 4% auto;">
-                  <h2 style="display: flex; justify-content: center; padding: 40px 15px 0px;">Loading Error</h2>
-                  <p style="display: flex; justify-content: center; padding: 10px 15px 40px;">
-                    ${message}
-                  </p>
-                </div>
-              </div>
-            </div>
-          </body>
-        </html>`);
+  const errorResponse = `<!DOCTYPE html>
+  <html>
+    <head>
+      <title>One App</title>
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <meta name="application-name" content="one-app">
+    </head>
+    <body style="background-color: #F0F0F0">
+      <div id="root">
+        <div>
+          <div style="width: 70%; background-color: white; margin: 4% auto;">
+            <h2 style="display: flex; justify-content: center; padding: 40px 15px 0px;">Loading Error</h2>
+            <p style="display: flex; justify-content: center; padding: 10px 15px 40px;">
+              ${message}
+            </p>
+          </div>
+        </div>
+      </div>
+    </body>
+  </html>`;
+
+  if (errorPage) {
+    safeSend(res, errorPage);
```

Comment:
if errorPage defaults to the value of errorResponse then you don't need this.

Reply:
The issue would be with `message` that we change depending on the status code right?

Reply:
Agree with @JAdshead with this small refactor if you want to apply it by just having 1 safeSend and the contents of the error change depending on the value of the `errorPageUrl` i.e. 

moving the logic of the default page to the top setting the value of errorPage to be the default, then overriding it if setErrorPage is there, and at the end do only one `safeSend(res, errorPage)` removing that if statement.

Reply:
You are right the if statement is required to determine if sending the custom or default error page... the default error page could be moved outside and pass the message as a parameter from `renderStaticErrorPage()` but probably not worth the hassle